site/source/tutorials/002_creating_an_application/index.md

```diff
@@ -28,58 +28,62 @@ You also need to be familiar with TypeScript as Dojo uses it extensively.
 
-{% include_codefile 'demo/initial/biz-e-corp/src/main.ts' lines:6-9 %}
+{% instruction 'Review these lines in `main.ts`:' %}
 
-{% aside "Why is it called a Projector?" %}
-A physical projector takes some form of media, such as slides or video, and projects them onto a surface, such as a wall. In a similar way, Dojo's projector takes a virtual representation of the application and projects it onto the actual page.
-{% endaside %}
+{% include_codefile 'demo/initial/biz-e-corp/src/main.ts' lines:4-6 %}
 
-These lines are the key to allowing the projector to coordinate between the virtual DOM and the rendered HTML that the user sees. The first line creates a class that registers the `HelloWorld` widget as the root widget, making it aware of the Dojo application. An instance is then created and its `append` method is used to make the projector aware of the HTML document.
+These lines are the key to allowing the projector to coordinate between the virtual DOM and the rendered HTML that the user sees. The final step is to call `mount` on the value returned from the `renderer` call. By default the `renderer` will mount the application on the HTML documents body, but this can be overridden by providing a `domNode` option to the `mount` function.
 
-Whenever a Dojo application needs to update the DOM, it informs the vdom system. The DOM updates do not, however, immediately re-render the page. Instead, it registers a render request with the application via the window's [requestAnimationFrame()](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) method. This method allows the application to delay updating the DOM until the browser is ready to re-render the page. By doing this, update operations can be grouped together which improves rendering performance. When the window re-renders the page, the application's current virtual DOM is used to determine what updates need to be made in a single operation. This allows the application to update the virtual DOM as often as required without the risk of reducing the application's responsiveness.
+{% instruction 'The `mount` operation in `main.ts`:' %}
+
+{% include_codefile 'demo/initial/biz-e-corp/src/main.ts' line:7 %}
 
 {% section %}
 
 ## Widgets
 
-Widgets are the basic building blocks of Dojo's user interface. They combine both the visual and behavioral aspects of a component into a single element. These aspects are encapsulated within the widget's implementation. The widget then exposes properties and methods that allow other components to interact with it. Consider the following diagrams:
+Up until now, we've return virtual DOM that represents HTML elements directly from the `renderer`, however that wouldn't scale very well as an application grow.
```

Comment:
Up until now, we've return virtual DOM that represents HTML elements directly from the `renderer`, however that wouldn't scale very well as an application grow. -> Until now, we have returned a virtual DOM that represents HTML elements directly from the `renderer`, however, that would not scale as an application grows.